### PR TITLE
tooltip contrast

### PR DESCRIPTION
**Pull request by PablitoGS**

### CODE REVIEW

File: html/tooltip.html [hunk 1]
```diff
@@ -1,42 +1,60 @@
...
+    </div>
+  </div>
+
+  <script>
+    $(document).ready(function() {
+      $(".tooltip-wrapper").toolTip();
+    });
+    $(window).resize(function() {
+      $(".tooltip-wrapper").toolTip();
+    });
```
Comment (beatrizip) [inline html/tooltip.html:56]: Repeat code

File: styles/components/tooltip.scss [hunk 2]
```diff
@@ -2,6 +2,59 @@
     $TOOLTIP
 \*------------------------------------*/
 
+@mixin tooltip($bgcolor){
+  @include font-rendering;
+  position: relative;
+  background-color: $bgcolor;
+  border: 1px solid #ccc;
```
Comment (beatrizip) [inline styles/components/tooltip.scss]: Without alphabetical order

File: styles/components/tooltip.scss [hunk 3]
```diff
@@ -2,6 +2,59 @@
...
+  border-radius: 4px;
+  box-shadow: 0 0 4px #ccc;
+  z-index: 9999;
+
+  &:before {
+    content:"";
+    position: absolute;
+    bottom: 100%;
```
Comment (beatrizip) [inline styles/components/tooltip.scss:18]: Without alphabetical order

File: styles/components/tooltip.scss [hunk 4]
```diff
@@ -2,6 +2,59 @@
...
+    height: 0;
+    border-bottom: 12px solid #ccc;
+    border-left: 12px solid transparent;
+    border-right: 12px solid transparent;
+  }
+
+  &:after {
+    content:"";
+    position: absolute;
+    bottom: 100%;
```
Comment (beatrizip) [inline styles/components/tooltip.scss]: Without alphabetical order

File: styles/components/tooltip.scss [hunk 5]
```diff
@@ -2,6 +2,59 @@
...
+    border-left: 10px solid transparent;
+	}
+}
+
+@mixin tooltip-layer($bgcolor) {
+	position: fixed;
+	overflow: auto;
+	top: 0;
+	bottom: 0;
+	background-color: $bgcolor;
```
Comment (beatrizip) [inline styles/components/tooltip.scss]: Without alphabetical order

File: styles/components/tooltip.scss [hunk 6]
```diff
@@ -2,6 +2,59 @@
...
+
+	@include mediaquery ($bp-tablet) {
+		position: absolute;
+		overflow: visible;
+		top: 34px;
+		left: -31px;
+		bottom: auto;
+		background-color: transparent;
```
Comment (beatrizip) [inline styles/components/tooltip.scss]: Without alphabetical order

File: styles/components/tooltip.scss [hunk 7]
```diff
@@ -20,26 +73,48 @@
...
 .close-tooltip {
-	display: inline-block;
+	@include iconfont-Close;
+	position: absolute;
+	top: 12px;
+	right: 5px;
+	left: auto;
+	width: 30px;
+	display: block;
 	overflow: hidden;
```
Comment (beatrizip) [inline styles/components/tooltip.scss]: Without alphabetical order

**Review by beatrizip — APPROVED** [review]

File: styles/components/tooltip.scss (hunk 2 above)
Comment (nguasch) [reply styles/components/tooltip.scss]: border has to be a var

File: styles/components/tooltip.scss [hunk 8]
```diff
@@ -2,6 +2,59 @@
...
+  @include font-rendering;
+  background-color: $bgcolor;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-shadow: 0 0 4px #ccc;
+  position: relative;
+  z-index: 9999;
+
+  &:before {
+    border-bottom: 12px solid #ccc;
```
Comment (nguasch) [inline styles/components/tooltip.scss]: border has to be a var

File: html/tooltip.html [hunk 9]
```diff
@@ -1,42 +1,60 @@
...
+      </div>
+
+      <!-- tooltip contrast -->
+      <div data-close="Cerrar" class="tooltip-wrapper">
+        <a class="js-tooltip-open iconfont-Info iconfont-18px alternative-action no-print" href="javascript:void(0);">
+			  <span class="hide">Info</span></a>
+        <div class="tooltip-layer tooltip-layer-contrast move-down-tooltip--mobile tooltip-fade-out">
+          <div class="tooltip tooltip-contrast inner" style="width: 480px;"><a title="Cerrar" class="close-tooltip" href="#">Cerrar</a>
+            <h3>Información de utilidad</h3>
+            <p>Algunas empresas prefieren no mostrar su nombre abiertamente en <a href="#">sus ofertas</a>. Los motivos pueden ser varios:</p>
```
Comment (nguasch) [inline html/tooltip.html]: `a` needs `link-underline` class to show underline on link

**Review by nguasch — APPROVED** [review]

**Pull request merged**